app/views/GermlineView/components/Report/index.tsx

```diff
@@ -111,6 +118,34 @@ const GermlineReport = ({
     }
   }, [report, history]);
 
+  const handleTSVExport = useCallback(() => {
+    const date = getDate();
+
+    gridApi.exportDataAsCsv({
+      suppressQuotes: true,
+      columnSeparator: '\t',
+      columnKeys: colApi.getAllDisplayedColumns()
+        .filter((col) => col.colDef.headerName !== 'Actions' && col.colDef.headerName)
+        .map((col) => col.colId),
+      fileName: `ipr_${report.patientId}_${report.ident}_germline_${date}.tsv`,
+      processCellCallback: (({ value }) => (typeof value === 'string' ? value?.replace(/,/g, '') : value)),
+    });
+  }, [colApi, gridApi, report]);
+
+  const handleMenuItemClick = useCallback((action) => {
```

Comment:
since this is only used by the export this function can be removed and the relevant logic of `setMenuAnchor(null);` added to  `handleTSVExport` instead